server/engine/telemetry.go

```diff
@@ -63,6 +63,8 @@ func PublishToSegment(db *gorm.DB) {
 	model.SetMetric(db, model.ApplicationId, config.GetEnvironment().APPLICATION_ID)
 	// time.RFC3339 format is the Go equivalent to ISO 8601 format (minus the milliseconds)
 	model.SetMetric(db, model.EndTime, time.Now().Format(time.RFC3339))
+	// time.RFC3339 format is the Go equivalent to ISO 8601 format (minus the milliseconds)
+	model.SetMetric(db, model.StartTime, config.GetEnvironment().START_TIME)
```

Comment:
If the value wasn't set I guess this will be empty, might want to check for empty and not include it in that case (that is what the comment on line 216 of environment.go is stating..

Reply:
should be good now